### ISSUE TITLE
[TESTNET BUG] Docker miner stuck at "stacks_tip_height" 0

### ISSUE THREAD

**Issue by pradel** [blockstack/stacks-blockchain#2002]
## Describe the bug

I am currently running 2 miners. One is running from the git repo, and is working fine. The other one is running via docker using the `blockstack/stacks-blockchain:v23.0.0.9-krypton` image. I also had the same issue with `blockstack/stacks-blockchain:v23.0.0.8-krypton`.
The docker miner is currently stuck at "stacks_tip_height" 0.
The endpoint `/v2/info` is currently returning

```
{"peer_version":385875968,"pox_consensus":"7e154334b56126211218b3fbd54a5d1fc7639948","burn_block_height":343,"stable_pox_consensus":"1e74fc75fad4b1be8fc958ad731053c00a35594d","stable_burn_block_height":342,"server_version":"blockstack-core 0.0.1 => 23.0.0.0 (, release build, linux [x86_64])","network_id":2147483648,"parent_network_id":3669344250,"stacks_tip_height":0,"stacks_tip":"0000000000000000000000000000000000000000000000000000000000000000","stacks_tip_consensus_hash":"0000000000000000000000000000000000000000","unanchored_tip":"4ec7ae4a471502547f69ff7165ce5eab0257def1a6607770512fb66e5fcc7e79","exit_at_block_height":null}
```

## Steps To Reproduce

Just follow the docker deployment tutorial on a new server. I tried 3 times on a new server and same problem on all 3 instances.

## Expected behavior

I expect the docker image to work the same as cloning the repo and start the miner manually.

## Environment

 - OS: Ubuntu 20.04
 - Docker version 19.03.13, build 4484c46d9d
 - Docker image: blockstack/stacks-blockchain:v23.0.0.9-krypton

## Additional context

Docker logs:

[docker.log](https://github.com/blockstack/stacks-blockchain/files/5439836/docker.log)

**Comment by diwakergupta** [comment]
@pradel can you try with the latest version? We believe this was addressed as part of #2014 -- we were not setting read timeouts on bitcoin sockets, which _might_ explain the behavior you saw.

**Comment by diwakergupta** [comment]
@pradel I'm going to close this out, please reopen if you're still seeing this.
